service/keyv3/key.go

```diff
@@ -37,6 +37,9 @@ const (
 	FlannelEnvPathPrefix = "/run/flannel"
 
 	K8SKVMHealthDocker = "quay.io/giantswarm/k8s-kvm-health:ddf211dfed52086ade32ab8c45e44eb0273319ef"
+
+	// default memory for k8s-kvm container.
+	defaultWorkerMemory = "1G"
```

Comment:
Not valid godoc. I'd like the godoc to express it's qemu overhead.

Reply:
Good point, fixed.